gui/default/syncthing/folder/editIgnoresModalView.html

```diff
@@ -14,12 +14,13 @@
     </dl>
   </div>
   <div class="modal-footer">
-    <div class="pull-left"><span translate>Editing</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code></div>
-    <button type="button" class="btn btn-primary btn-sm" ng-click="saveIgnores()" data-dismiss="modal">
+    <div class="pull-left" ng-show="editingExisting"><span translate>Editing</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code></div>
+    <div class="pull-left" ng-show="!editingExisting"><span translate>Creating ignore patterns, overwriting an existing file at</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code>.</div>
```

Comment:
I know this is now a sad never ending story so it hurts a bit to bring this up now, but this string should really not assume that the path is the last word of the sentence and has a period after it. Some languages may require a different sentence structure so the path should be a placeholder.

(The "Editing <foo>" thing is a historical accident.)